src/main/java/net/sf/mpxj/ManuallyScheduledTaskCalendar.java

```diff
@@ -0,0 +1,118 @@
+/*
+ * file:       ProjectCalendar.java
+ * author:     Fabian Schmidt
+ * date:       16/08/2024
+ */
+
+/*
+ * This library is free software; you can redistribute it and/or modify it
+ * under the terms of the GNU Lesser General Public License as published by the
+ * Free Software Foundation; either version 2.1 of the License, or (at your
+ * option) any later version.
+ *
+ * This library is distributed in the hope that it will be useful, but
+ * WITHOUT ANY WARRANTY; without even the implied warranty of MERCHANTABILITY
+ * or FITNESS FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public
+ * License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with this library; if not, write to the Free Software Foundation, Inc.,
+ * 59 Temple Place, Suite 330, Boston, MA 02111-1307, USA.
+ */
+
+package net.sf.mpxj;
+
+import java.time.DayOfWeek;
+import java.time.LocalDate;
+import java.time.LocalDateTime;
+import java.time.LocalTime;
+
+/**
+ * This class represents a Calendar Definition record for an MPP Manually Scheduled task.
+ */
+public class ManuallyScheduledTaskCalendar extends ProjectCalendar
+{
+   public ManuallyScheduledTaskCalendar(ProjectCalendar calendar, ResourceAssignment assignment)
+   {
+      super(calendar.getParentFile(), true);
+      m_calendar = calendar;
+      m_assignment = assignment;
+      this.setParent(calendar);
```

Comment:
```suggestion
```

You can remove this, we don't need to create a parent-child relationship between the calendars.

Reply:
I removed the `setParent` operation.